CMakeLists.txt

```diff
@@ -75,6 +76,60 @@ if (GLOW_WITH_OPENCL)
   find_package(OpenCL REQUIRED)
 endif ()
 
+# Returns the name of the backend enabling
+MACRO(getBackendEnableVariable result backend_dir_name)
+  string(TOUPPER "GLOW_WITH_${backend_dir_name}" ${result})
+ENDMACRO()
+
+# Return in result the name of curdir sub-directories
+MACRO(getSubDirList result curdir)
+  file(GLOB children RELATIVE ${curdir} ${curdir}/[^\.]*)
+  SET(dirlist "")
+  FOREACH(child ${children})
+    IF(IS_DIRECTORY ${curdir}/${child})
+      LIST(APPEND dirlist ${child})
+    ENDIF()
+  ENDFOREACH()
+  SET(${result} ${dirlist})
+ENDMACRO()
+
+# External backends
+set(EXTERNAL_BACKENDS_DIR ${GLOW_SOURCE_DIR}/externalbackends)
+include_directories(${EXTERNAL_BACKENDS_DIR})
+getSubDirList(SUBDIRS ${EXTERNAL_BACKENDS_DIR})
+
+FOREACH(child ${SUBDIRS})
```

Comment:
can you place that to a separate file GlowExternalBackends.cmake in cmake/modules and include it here.

Reply:
Done. I've moved the external backend related cmake code in cmake/modules